src/pages/GetInvolvedPage.tsx

```diff
@@ -1,5 +1,36 @@
+import { Map, MapGeoJsonSource, MapContainer } from "@/atoms/MapAtoms";
+import { Theme } from "@/theme/Theme";
+import  BPDA from "@/data/BPDA";
+import { Marker } from "react-map-gl";
+
+const BPDADevelopmentLayer = () => (
+  <MapGeoJsonSource
+    data={BPDA}
+    id="BPDA"
+    type="circle"
+    color={Theme.colors.blue}
+  />
+);
+
 export default function GetInvolvedPage() {
   return (
-    <h1>Get Involved</h1>
-  )
+    <MapContainer>
+      <Map mapStyle={Theme.map.dark}>
+        <>
+          <BPDADevelopmentLayer />
+          {BPDA.map((data:any) => (
+            <Marker
+              latitude={data.features.type.coordinates[0]}
+              longitude={data.features.type.coordinates[1]}
+              offsetLeft={-20}
+              offsetTop={-10}
+            >
+              <div>You are here</div>
+            </Marker>
```

Comment:
Something to also consider is to create an in-house `MapMarkerGeoJson` component (or something of the like) that glues together a layer and a 'style' - take a look here in the react-map-gl example and see how they created their own `Pin` component that kind of glues together an SVG paired with Marker component and that they are bound together through the `id` attribute